src/test/kotlin/net/revanced/patcher/ReaderTest.kt

```diff
@@ -1,12 +1,13 @@
 package net.revanced.patcher
 
+import com.sun.xml.internal.messaging.saaj.util.ByteOutputStream
```

Comment:
Wrong import. Don't ever use classes from `com.sun.*`, it's recommended against by Oracle since these classes are implementation-bound. (this class only exists in certain JVMs, definitely not in Android at least)
```suggestion
import java.io.ByteArrayOutputStream
```